### ISSUE TITLE
Could not find resource files_external/js/main.js to load

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/server#41409]
```json
{
  "reqId": "TCmV3pFoYfCJSQKvYQR4",
  "level": 3,
  "time": "2023-11-13T09:51:17+00:00",
  "remoteAddr": "…",
  "user": "…",
  "app": "jsresourceloader",
  "method": "GET",
  "url": "/apps/files/files/…?dir=/",
  "message": "Could not find resource files_external/js/main.js to load",
  "userAgent": "Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/119.0",
  "version": "28.0.0.4",
  "data": {
    "app": "jsresourceloader"
  }
}
```

Triggered in https://github.com/nextcloud/server/blob/a5a8655bebb8652206ea5aa886699f706739e1fe/apps/files_external/lib/Listener/LoadAdditionalListener.php#L55 but the file is non existent in the repo:
https://github.com/nextcloud/server/tree/38480fda3cd1f10652bc1e854207b074921e66b8/apps/files_external/js

**Comment by szaimen** [comment]
cc @nextcloud/server-frontend

**Comment by nickvergessen** [comment]
Fixed already by #41766
